build/convert/netdef_to_onnx/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.6-slim
+ENV LC_ALL="C.UTF-8" \
+  LANG="C.UTF-8"
+
+ARG ORMB_VERSION=0.0.8
+ARG ORMB_TAG=v${ORMB_VERSION}
+ARG ORMB_TAR_FILENAME=ormb_${ORMB_VERSION}_Linux_x86_64.tar.gz
+
+ENV http_proxy=10.20.47.147:3128 
+ENV https_proxy=10.20.47.147:3128 
```

Comment:
ENV http_proxy=10.20.47.147:3128
ENV https_proxy=10.20.47.147:3128

fixed ip for what?

Reply:
fixed